<ui version="4.0" >
 <author>Aaron Racicot</author>
 <class>SelectGear</class>
 <widget class="QDialog" name="SelectGear" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Select Gear</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="title" >
      <string>Fishery Sector</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="fishery_sector_label" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="fishery_text" >
        <property name="text" >
         <string>Select Gear Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gear_comboBox" >
        <property name="maximumSize" >
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Hook and Line</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Gillnets</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Hooka (compressor)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Traps</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Trawling</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Purse Seine</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Other</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget_2" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="pbnStepFinished" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnStartShapes" >
        <property name="text" >
         <string>Start Drawing Shapes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
